code/decision_transformer/decision_transformer.py

```diff
@@ -109,6 +111,32 @@ def get_action(self, states: torch.Tensor, actions: torch.Tensor, rtgs: torch.Te
         Returns:
             torch.Tensor: predicted actions
         """
-        _, action_preds, _ = self.forward(states, actions, None, rtgs, timesteps, mask)
+        # normalize given actions so it would fit the DT engine
+        if self.action_min is not None and self.action_max is not None:
+            actions = self.normalize_action(actions)
+
+        _, action_preds, _ = self.forward(states, actions, rtgs, timesteps, mask)
+
+        # unnormalize the action prediction before returning it
+        if self.action_min is not None and self.action_max is not None:
+            action_preds = self.unnormalize_action(action_preds)
+
+        return action_preds[0,-1]
+
+    def normalize_action(self, action):
+        """
+        :param action: given action to normalize
+        :return: the normalized action
+        """

```

Comment:
What is this function format?
why not like the others?
```suggestion
        """
        Args:
            action: given action to normalize
        Returns:
            normalized action
        """
```

Reply:
Fixed.